rskj-core/src/main/java/co/rsk/remasc/Remasc.java

```diff
@@ -129,24 +126,7 @@ void processMinersFees() throws IOException, BlockStoreException {
         feesPayer.payMiningFees(processingBlockHeader.getHash().getBytes(), payToRskLabs, remascConstants.getRskLabsAddress(), logs);
         fullBlockReward = fullBlockReward.subtract(payToRskLabs);
 
-        // TODO to improve
-        // this type choreography is only needed because the RepositoryTrack support the
-        // get snapshot to method
-        Repository processingRepository = ((RepositoryTrack)repository).getOriginRepository().getSnapshotTo(processingBlockHeader.getStateRoot());
-        // TODO to improve
-        // and we need a RepositoryTrack to feed RemascFederationProvider
-        // because it supports the update of bytes (notably, RepositoryImpl don't)
-        // the update of bytes is needed, because BridgeSupport creation could alter
-        // the storage when getChainHead is null (specially in production)
-        processingRepository = processingRepository.startTracking();
```

Comment:
Isn't it possible for the repository to not be tracking here?

Reply:
Yes, I removed this code and it's not tracking anymore.
Am I missing something?